docs/package.json

```diff
@@ -42,7 +42,7 @@
 		"@fluid-tools/api-markdown-documenter": "^0.9.0",
 		"@fluid-tools/markdown-magic": "file:../tools/markdown-magic",
 		"@fluidframework/build-common": "^2.0.0",
-		"@microsoft/api-extractor-model": "^7.27.3",
+		"@microsoft/api-extractor-model": "7.27.6", 
```

Comment:
Remind me why this change?

Reply:
[Pipeline Failure: Build site artifact](https://github.com/microsoft/FluidFramework/actions/runs/6642240635/job/18046543552?pr=17651)

--

There was a bug in the `api-extractor` which created the pipeline failure above. @Josmithr suggested installing `"@microsoft/api-extractor/model": "7.27.6"` instead to bypass the error. 

It's been a while since I made changes to this PR, so I can check if the build passes after reverting this change.